Description: Remove explicit checks for stub libraries no longer in GLSL
Author: Peter J. Mello <admin@petermello.net>
Origin: other, https://git.launchpad.net/~libretro/libretro/+git/RetroArch-debian/commit/patches/0004-remove-GLSL-library-checks.patch?id=5491e645fa1b77ba22cac8673980fb1fc36acbbe
Bug-Debian: https://bugs.debian.org/1064681
Forwarded: yes, https://github.com/libretro/RetroArch/issues/16571
Last-Update: 2026-09-06

--- a/qb/config.libs.sh
+++ b/qb/config.libs.sh
@@ -843,16 +843,13 @@ if [ "$HAVE_GLSLANG" != no ]; then
 
    check_lib cxx GLSLANG -lglslang '' '-lSPIRV'
    check_lib cxx GLSLANG_OSDEPENDENT -lOSDependent
-   check_lib cxx GLSLANG_OGLCOMPILER -lOGLCompiler
    check_lib cxx GLSLANG_MACHINEINDEPENDENT -lMachineIndependent
    check_lib cxx GLSLANG_GENERICCODEGEN -lGenericCodeGen
-   check_lib cxx GLSLANG_HLSL -lHLSL '' '-lglslang -lSPIRV'
    check_lib cxx GLSLANG_SPIRV -lSPIRV
    check_lib cxx GLSLANG_SPIRV_TOOLS_OPT -lSPIRV-Tools-opt
    check_lib cxx GLSLANG_SPIRV_TOOLS -lSPIRV-Tools
 
    if [ "$HAVE_GLSLANG" = no ] ||
-      [ "$HAVE_GLSLANG_HLSL" = no ] ||
       [ "$HAVE_GLSLANG_SPIRV" = no ] ||
       [ "$HAVE_GLSLANG_SPIRV_TOOLS_OPT" = no ] ||
       [ "$HAVE_GLSLANG_SPIRV_TOOLS" = no ]; then
